vault/hooks/run

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -xe
+
+exec 2>&1
+
+exec vault server -dev-listen-address=0.0.0.0:8200 -dev
```

Comment:
We probably don't want to hardcode the listen port/addr here. This should be exposed as a configurable